public/popup.css

```diff
@@ -1,236 +1,275 @@
 :root {
+  --sb-main-font-family: "Source Sans Pro", sans-serif;
   --sb-main-bg-color: #222626;
-  --sb-main-fg-color: white;
-  --sb-gray-fg-color: #444848;
-  --sb-on-white-bg: black;
-  --sb-green-bg: #077B27;
+  --sb-main-fg-color: #fff;
+  --sb-grey-bg-color: #333;
+  --sb-red-bg-color: #cc1717;
 }
 
-#sponsorBlockPopupHTML {
-  color-scheme: dark;
+/*
+ * Default browser styles
+ */
+
+html {
+  box-sizing: border-box;
 }
 
-#sponsorBlockPopupBody .hidden {
-  display: none !important;
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
 }
 
-.sponsorTimesCategoryColorCircle {
-  margin: 0 8px;
+p {
+  margin: 0;
 }
 
-.voteButtonsContainer--hide {
-  display: none;
+a {
+  color: var(--sb-main-fg-color);
+  text-decoration: none;
 }
 
-@media only screen and (max-width: 600px) {
-  #sponsorBlockPopupBody {
-    width: 100%; 
-  }
+button {
+  background: transparent;
+  border: 0;
+  cursor: pointer;
+  padding: 0;
```

Comment:
It's not possible to have generic selectors like this as this css is used on YouTube for the popup on the webpage, so the CSS can't interfere.